testutil/testnode/full_node.go

```diff
@@ -197,7 +197,7 @@ func DefaultNetwork(t *testing.T, blockTime time.Duration) (accounts []string, c
 	genState, kr, err := DefaultGenesisState(accounts...)
 	require.NoError(t, err)
 
-	tmNode, app, cctx, err := New(t, DefaultParams(), tmCfg, false, genState, kr)
+	tmNode, app, cctx, err := New(t, DefaultParams(), tmCfg, false, genState, kr, tmrand.Str(6))
```

Comment:
we could also bubble this up if you all want to hopefully avoid duplicate code in https://github.com/celestiaorg/celestia-node/blob/main/core/testing.go, but I understand just using the current apis